dashboard/db/schema.rb

```diff
@@ -1056,7 +1057,7 @@
     t.index ["user_id", "script_id", "level_id"], name: "index_puzzle_ratings_on_user_id_and_script_id_and_level_id", unique: true, using: :btree
   end
 
-  create_table "queued_account_purges", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci" do |t|
+  create_table "queued_account_purges", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci" do |t|
```

Comment:
is the change from utf8 to utf8mb4 expected? are there other places we would also expect that?

Reply:
Hmm that's an interesting point. This might explain the problem